flake.nix

```diff
@@ -61,29 +63,76 @@
           name = "ihaskell";
           returnShellEnv = false;
           modifier = pkgs.haskell.lib.dontCheck;
-          overrides = (mkEnv hsPkgs).ihaskellOverlay ;
-          withHoogle = true;
+          overrides = (mkEnv hsPkgs).ihaskellOverlay;
+          withHoogle = false;
         };
 
     in {
 
       packages = {
         ihaskell-dev = mkDevShell ghcDefault;
-        ihaskell-dev-921 = mkDevShell ghc921;
-        ihaskell-dev-8107 = mkDevShell ghc8107;
-        ihaskell-dev-884 = mkDevShell ghc884;
+        ihaskell-921-dev = mkDevShell ghc921;
+        ihaskell-8107-dev = mkDevShell ghc8107;
 
         ihaskell = mkExe ghcDefault;
         ihaskell-8107 = mkExe ghc8107;
         ihaskell-921 = mkExe ghc921;
 
         # I actually wish those would disappear ? let jupyterWith or use deal with it
         ihaskell-env = mkEnv ghcDefault;
-        ihaskell-env-8107 = mkEnv ghc8107;
+        ihaskell-8107-env = mkEnv ghc8107;
+        ihaskell-921-env = mkEnv ghc921;
       };
 
       defaultPackage = self.packages.${system}.ihaskell;
 
-      devShell = self.packages.${system}.ihaskell-dev;
-    });
+      devShells = {
+        ihaskell-dev = mkDevShell ghcDefault;
+        ihaskell-8107 = mkDevShell ghc8107;
+        ihaskell-921 = mkDevShell ghc921;
+      };
+    }) // {
+
+      overlay = final: prev: {
+        python3 = prev.python3.override {
+          packageOverrides = pfinal: pprev: {
+            openapi-core = pprev.openapi-core.overridePythonAttrs(oa: {
+
+              doCheck = false;
+              doInstallCheck = false;
+            });
+          };
+        };
+
+        ihaskellGhc921 = prev.haskell.packages.ghc921.extend (final: prev: let 
```

Comment:
Also there's a reason we use `.override` in `release.nix` instead of `.extend` even though the latter is more convenient: https://github.com/NixOS/nixpkgs/issues/26561#issuecomment-354862897

Reply:
good point